examples/webgl_lines_fat.html

```diff
@@ -223,6 +226,56 @@
 
 			//
 
+			function onDocumentMouseMove( event ) {
+
+				if ( ! interactive ) return;
+
+				event.preventDefault();
+
+				var intersects = [];
```

Comment:
Now, you are instantiating an array every mouse move. That defeats the purpose. You will need to create one array in a closure and reuse it. See https://github.com/mrdoob/three.js/blob/dev/docs/api/en/core/Raycaster.html#L151.

Like I said elsewhere, others may have varying opinions on the usefulness of doing this.

/ping @Mugen87 for opinion